### PR TITLE
Array/exercise 13

### PR DESCRIPTION
**Pull request by ntnghia0320**

### CODE REVIEW

File: src/main/java/ArrayExercise13.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+public class ArrayExercise13 {
+    public int[][] deleteXY(int[][] matrix, int x, int y) {
```
Comment (voquanghoa) [inline src/main/java/ArrayExercise13.java]:
```suggestion
    public int[][] deleteRowColumn(int[][] matrix, int colmn, int row) {
```

File: src/main/java/ArrayExercise13.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        int m = matrix.length;
+        int n = matrix[0].length;
```
Comment (voquanghoa) [inline src/main/java/ArrayExercise13.java]:
```suggestion
        int rows = matrix.length;
        int columns = matrix[0].length;
```

File: src/main/java/ArrayExercise13.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        int l = 0;
+
+        int[][] matrixResult = new int[m - 1][n - 1];
+        for (int i = 0; i < m; i++) {
+            if (i == x - 1) continue;
+            for (int j = 0; j < n; j++) {
+                if (j == y - 1) continue;
+                matrixResult[k][l] = matrix[i][j];
+
+                l++;
```
Comment (voquanghoa) [inline src/main/java/ArrayExercise13.java]:
```suggestion
                int tempRow =  row <= rowToDelete? row : row - 1;
                int tempColum =  colum <= columnToDelete? colum : colum  - 1;
                 
                matrixResult[tempRow ][tempColum ] = matrix[row ][colum];
```

File: src/test/java/ArrayExercise13Test.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class ArrayExercise13Test {
+    ArrayExercise13 arrayExercise13 = new ArrayExercise13();
+
+    @Test
+    public void test_deleteXY() {
+        assertArrayEquals(arrayExercise13.deleteXY(new int[][]{
```
Comment (voquanghoa) [inline src/test/java/ArrayExercise13Test.java]: Sai sai

File: src/main/java/ArrayExercise13.java (hunk 1 above)
Comment (ntnghia0320) [reply src/main/java/ArrayExercise13.java]: Done

File: src/main/java/ArrayExercise13.java (hunk 2 above)
Comment (ntnghia0320) [reply src/main/java/ArrayExercise13.java]: Done

File: src/main/java/ArrayExercise13.java (hunk 3 above)
Comment (ntnghia0320) [reply src/main/java/ArrayExercise13.java]: Done

File: src/test/java/ArrayExercise13Test.java (hunk 4 above)
Comment (ntnghia0320) [reply src/test/java/ArrayExercise13Test.java]: Done

**Pull request merged**